src/elements/content-picker/Footer.js

```diff
@@ -102,6 +102,7 @@ const Footer = ({
                                 isDisabled={!selectedCount}
                                 onClick={onChoose}
                                 type="button"
+                                disabled={!selectedCount}
```

Comment:
we might want to make an `isChooseButtonDisabled` variable now. i see three usages

Reply:
yarp